src/Index/page.jsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Introduction from '../components/Introduction';
 import { ReduxIndexButtonBar, ReduxIndexBarChart } from './reduxer';
 import dictIcons from '../img/icons';
 import { components } from '../params';
-import getProjectsList from './relayer';
+import getProjectNodeCounts from './utils';
 import './page.less';
 
 class IndexPageComponent extends React.Component {
   constructor(props) {
     super(props);
-    getProjectsList();
+    getProjectNodeCounts((res) => {
```

Comment:
I think that this should go in `componentDidMount`, not the constructor.
https://reactjs.org/docs/react-component.html#constructor